src/core/search/searchapi.js

```diff
@@ -242,4 +242,36 @@ export default class SearchApi {
     return request.get()
       .then(response => response.json());
   }
+
+  /**
+ * Initiate a search
```

Comment:
I think your indentation is a little off compared to the other functions

Reply:
I will make sure to stick to semistandard regulations!